blocks/footer/footer.js

```diff
@@ -1,26 +1,30 @@
-// import { readBlockConfig, decorateIcons } from '../../scripts/lib-franklin.js';
+import { decorateIcons, getMetadata } from '../../scripts/lib-franklin.js';
```

Comment:
Thank you, @bstopp for explaining the footer js code and how this mechanic is supposed to work.  I appreciate the assist.